Dockerfile.build

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/centos/centos:stream8
+
+COPY --from=ghcr.io/yannh/kubeconform:v0.4.14 /kubeconform /usr/bin
+
+RUN --mount=type=tmpfs,destination=/var/cache\
+    --mount=type=tmpfs,destination=/root/.cache\
+    --mount=type=cache,target=/var/cache\
+    --mount=type=cache,target=/root/.cache\
+    dnf upgrade -y && dnf install -y make
```

Comment:
the upgrade seems redundant, no? unless I'm missing something

Reply:
I mean, we don't control here the amount of time spent on upgrades (might be more in a few weeks if repositories advance and the base image doesn't change) so unless it's not required I think it's better to remove it